src/test/groovy/graphql/parser/ParserTest.groovy

```diff
@@ -727,7 +729,7 @@ triple3 : """edge cases \\""" "" " \\"" \\" edge cases"""
         def input = "{,\r me\n\t} ,\n"
 
         when:
-        Document document = new Parser().parseDocument(input)
+        Document document = new Parser().parseDocument(input, true)
```

Comment:
Again the diff does not show it but this is the old ignored chars test - now needs to be ON explcitly